### DISCUSSION TITLE
节点无法激活

### DISCUSSION THREAD

**Discussion by zoukz115** [mangodager/smartx-dotnet#15] [General]
下午节点掉线了，现在同步到最新高度，版本为4.2.0，仍然无法上线。
![image](https://user-images.githubusercontent.com/71767976/172374437-83ddc73a-9ef3-4b52-b069-e9324a60c936.png)

**Comment by mangodager** [comment]
一般需要等一段时间，大概10-30分钟，系统会自动beruler，或者自己手动输入beruler命令

**Comment by mangodager** [comment]
把您的地址发一下，如果是多网卡， 设置一下config目录配置 publicIP。
![微信图片_20220608123822](https://user-images.githubusercontent.com/46709419/172532412-738505b8-6dbb-45e8-939c-e8c63d8bb5c5.jpg)

**Comment by 854771076** [comment]
回滚一下

**Comment by 854771076** [comment]
在节点的 console.html 页面下输入   MergeChain 密码
命令执行完成最好重启一下节点

**Comment by yuop2002cn** [comment]
此问题已解决
